pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -292,6 +292,40 @@ class Build {
 
         return jdkRepo
     }
+
+    private String allocateEBCnodesForTest(testType) {
+        def test_target = buildConfig.TARGET_OS + '_' + buildConfig.ARCHITECTURE
+        // All targets are not supported via EBC yet. We need to test that target and add to default.json later on to make it enable
+        if ( ! DEFAULTS_JSON['testDetails']['ebcEnabledTargets'][test_target].find { it == testType } ) {
+            context.println "EBC does not support for $test_target->$testType yet!"
+            return ''
+        }
+
+        // context.library 'jenkins-ci-websphere' 
+        def group_label_UUID = 'Adopt-semeru_test_machine_' + UUID.randomUUID().toString()
+        def num_machines = '0'
+        def test_index = DEFAULTS_JSON['testDetails']['defaultDynamicParas']['testLists'].indexOf(testType)
+        num_machines = DEFAULTS_JSON['testDetails']['defaultDynamicParas']['numMachines'].get(test_index)
+        def nodes_ebc_args = new Object[num_machines]
+
+        if (num_machines == '0' ) {
+            num_machines = '1'
+        }
+
+        def jobName = '/EBC_Create_Node'
+        def ebcJob = context.build job: jobName,
+            propagate: false,
+            wait: true,
+            parameters: [
+                    context.string(name: 'group_label', value: group_label_UUID),
+                    context.string(name: 'ebcPlan', value: 'hur-runtimesJenkins-ubuntu22_x86.yml'),
+                    context.string(name: 'ebcShortlist', value: ''),
+                    context.string(name: 'NUM_MACHINES', value: num_machines),
+                    context.string(name: 'TIME_LIMIT', value: '3') //TODO: use estimated time via default.json instead
```

Comment:
Assuming this is hours? 3 likely won't be enough. I think we can just set a max value here. Where do we return the nodes?

Reply:
I think that value have to be provided as an additional param in default.json. I do not have those numbers. Also setting a MAX for whole is a waste of resource.

Reply:
At the time we rely on EBC timeout which is not optimal. But we can add a trigger at the end of AQA test to destroy nodes too. I can work on it as an improvement.

Reply:
3 hours will be enough for sanity.perf but we'll have to increase this later.